docs/source/advanced_features/fusion_dataset/fusion_dataset_structure.rst

```diff
@@ -33,7 +33,9 @@ The TensorBay fusion dataset format looks like::
 Fusion dataset is the topmost concept in TensorBay format.
 Each fusion dataset includes a catalog and a certain number of fusion segments.
 
-The corresponding class of fusion dataset is :class:`~tensorbay.dataset.dataset.FusionDataset`.
+The corresponding class of fusion dataset is 
```

Comment:
The fusion_dataset_structure.rst is already hidden, there is no need to hide this part.